app/views/layouts/application.html.erb

```diff
@@ -5,15 +5,18 @@
     <meta name="viewport" content="width=device-width,initial-scale=1">
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
-
     <%= stylesheet_link_tag "application" %>
+    <%= render 'layouts/header' %>
   </head>
-
   <body class="govuk-template__body govuk-body">
     <div class="govuk-width-container">
       <main class="govuk-main-wrapper">
+      <% unless current_page?(root_path) %>
+        <%= render 'layouts/back-button' %>
```

Comment:
Don't use `layouts/` for things that are not layouts, it's very confusing. Most projects have a `shared` directory for partials that are shared.

Reply:
Good point, I wasn't sure on this. Will fix now